spec/name_format_validator_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'factories/profiles.rb'
+
+describe NameFormatValidator do
+
+  let(:subject) { NameFormatValidator }
+
+  let( :attribute ) { :first_name }
+  let (:object) { Profile.new }
```

Comment:
has a space between let and the parenthesis. Not a big deal, just for the sake of consistency.